wordpress-mobile/WordPress-iOS
# Jetpack: Change My Sites tab bar icon

## Requirement 

> Update main bottom tab bar icons (new icon set) - @osullivanchris 

Source pbArwn-5Uc-p2#comment-7395

## Screenshots

| Before | After |
| ------- | ----- |
| ![](https://user-images.githubusercontent.com/9609223/224656440-352ec16a-8a0e-4633-93ed-fc9f1d0c105e.png) | ![](https://user-images.githubusercontent.com/9609223/224655489-bd2a172a-660a-4034-95d8-124c049d67a3.png) |

## Test Instructions

1. Install Jetpack and log into your account.
2. **Verify** the My Site tab bar icon is the Jetpack logo.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
